mlflow/protos/model_registry.proto

```diff
@@ -529,8 +529,8 @@ message SearchModelVersions {
   // with string values wrapped in single quotes.
   optional string filter = 1;
 
-  // Maximum number of models desired. Max threshold is 200K.
-  optional int64 max_results = 2 [default = 200000];
+  // Maximum number of models desired. Default is 100. Max threshold is 1000.
+  optional int64 max_results = 2 [default = 100];
```

Comment:
Curious why we made this change

Reply:
I thought the previous default value is set to 200k just as a placeholder? I followed the same default value as registered models.

Reply:
How about 1000 (default) and 10000 (max)?